.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
I think the `GH_TOKEN` is not needed in this case, since `slate` isn't using any `@typeform` dependency

Reply:
you are right! good catch 🙇🏽‍♂️